src/pages/CDP.js

```diff
@@ -244,8 +254,22 @@ function CDPView({ cdpId }) {
       ) : (
         <LoadingLayout />
       ),
-    [cdp, showSidebar, cdps, account]
+    [cdp, cdpId, showSidebar, account]
+  );
+}
+
+function calcFromSdk(cdp) {
+  const collateralAmount_ = collateralAmount(
+    cdp.currency,
+    BigNumber(cdp.ink).shiftedBy(18)
```

Comment:
As you can see here, I had to undo the conversions performed in multicall transform functions (that is, the functions in the `returns` sections)--probably we want to keep completely raw values in the store

Reply:
👍